src/Features/Core/Portable/AddImport/Remote/IRemoteAddImportService.cs

```diff
@@ -0,0 +1,28 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Immutable;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.Packaging;
+using Microsoft.CodeAnalysis.Remote;
+using Microsoft.CodeAnalysis.SymbolSearch;
+using Microsoft.CodeAnalysis.Text;
+
+namespace Microsoft.CodeAnalysis.AddImport
+{
+    internal interface IRemoteAddImportService
```

Comment:
note, the name FeatureService was actually intentional.  in roslyn we have a few services realted to AddImport, the 'Feature Services' are the ones specific to the actual lightbulb/codeaction feature itself.  i.e. computing whihc imports to add.  the normal Services are about actually doign the adding.  i.e. the raw mechanical translation of how imports should be added to the tree.

can we restore the 'Feature' part of this name?  thanks!

Reply:
you could also call this the IRemoteAddImportCodeActionService if that would be clearer.

Reply:
IRemoteAddImportCodeActionService is definitely better. "FeatureService" are two very general names that don't add any information of the purpose.

Reply:
Renamed to IRemoteMissingImportDiscoveryService